### PR TITLE
Add volume support to cli

### PR DESCRIPTION
**Pull request by aasutossh**
`rustyvibes <volume> <soundpack_path>`

Maybe need to update the readme too?
Either update the source in cargo or instruction to build it locally.

### CODE REVIEW

**Comment by aasutossh** [comment]
closes #22 #20

**Comment by KunalBagaria** [comment]
Thank you so much for the PR, let me review it this week and merge.

**Comment by aasutossh** [comment]
let me know if anything needs to be improved.
have never used rust before.

File: src/play_sound.rs [hunk 1]
```diff
@@ -27,7 +29,8 @@ pub mod sound {
         tx
     }
 
-    pub fn play_sound(name: String) {
+    pub fn play_sound(name: String, input_volume: f32) {
+        unsafe { VOLUME = input_volume };
```
Comment (KunalBagaria) [inline src/play_sound.rs:33]: Let's not use unsafe code

File: src/play_sound.rs [hunk 2]
```diff
@@ -51,6 +54,7 @@ pub mod sound {
                         .clone()
                 };
                 let sink = rodio_wav_fix::Sink::try_new(&stream_handle).unwrap();
+                unsafe { sink.set_volume(VOLUME) };
```
Comment (KunalBagaria) [inline src/play_sound.rs:57]: Will need to remove unsafe blocks

**Review by KunalBagaria — CHANGES_REQUESTED** [review]

**Comment by aasutossh** [comment]
Could you provide some insights on how to achieve those actions without `unsafe`?

**Comment by KunalBagaria** [comment]
You can refer to #25

**Pull request closed without merging**